### ISSUE TITLE
CSS externel file not working

### ISSUE THREAD

**Issue by asfixcom** [opensupports/opensupports#832]
Hi,
I was made that
<link rel="stylesheet" href="css/global.css"> in index.php inside head section.

Inside global.css i put:
.card_red {
  background-color:#3f9d68
}
but no changes.
How to change something in styles?

**Comment by gunnerschile** [comment]
Maybe u need make changes in /bundle.js

**Comment by asfixcom** [comment]
Yes i can make changes in bundle.js  but what if there will be update? after update all my changes will gone

**Comment by gunnerschile** [comment]
Yes, when it is updated everything returns to the default values ... You must make the changes again when it is updated.

**Comment by asfixcom** [comment]
Any other method? other js file pinned to index.php that include css like in bundle? but i don't known how to make js file with css and no warranty that will be working
